### DISCUSSION TITLE
rename docker-compose.yml

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#2630] [Ideas] [answered]
It seems like `docker-compose.yml` should be named `compose.yaml`: https://docs.docker.com/compose/compose-file/03-compose-file, also the `version: ` is deprecated

**Comment by szaimen** [comment]
Hey, thanks for the heads-up! Would you mind creating a PR that fixes this? :)

**Reply by Zoey2936** [reply, marked as answer]
done #2643
